npe2/_from_npe1.py

```diff
@@ -23,6 +23,7 @@
 
 from npe2.manifest import PluginManifest
 from npe2.manifest.commands import CommandContribution
+from npe2.manifest.schema import ENGINE_VERSION
```

Comment:
why do we need to import and use this here if it's also the default value for engine?

Reply:
We want to have a default behavior for read, but when we're writing I think it's good to be opinionated: label the converted manifest with the version used to covert it.

Reply:
ahh... and when we serialize it excludes not-set right?

Reply:
I guess the other part of the answer is I imagine we may not always have a default value for 'engine' or the default behavior may change. I'm guessing we always want to write the engine version though.

Reply:
I totally agree we always want to write the engine, but i was confused why it goes in the constructor here (since it will result in an identical PluginManifest even if you excluded it).

Just so you know, since this is more of a serialization issue than an instantiation issue, another option here would be to use the `include=['engine']` parameter when serializing with `PluginManifest.json()`

Reply:
ah ok. I could go either way. I'm probably overthinking it. I'll leave it as is and we can revisit if it comes up